style.css

```diff
@@ -89,3 +139,705 @@ body {
   display: flex;
   justify-content: space-around;
 }
+
+.social-lg {
+  display: none;
+}
+
+.recent-works {
+  gap: 24px;
+  box-sizing: border-box;
+}
+
+img {
+  width: 100%;
+}
+
+.work {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.desk-img {
+  margin: 0 12px;
+}
+
+.mobile-img {
+  max-width: 327px;
+  display: flex;
+  justify-content: center;
+}
+
+.title-header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 307px;
+  flex-direction: column;
+}
+
+.title-border {
+  border: 1px solid #212121;
+  min-width: 327px;
+}
+
+.title {
+  height: 83px;
+  font-family: Crete RoundArial, Arial, Helvetica, sans-serif;
```

Comment:
Good job using the fonts from the provided design! :clap:  However, the specified font is not being applied because is missing the quotes like so: `font-family: "Crete"` :exclamation: